.github/workflows/scripts/prs/detectTargetBranch.js

```diff
@@ -22,90 +22,63 @@ module.exports = async ({ core, context, github }) => {
-        core.setOutput('TRANSFER_LABELS', transferLabels.join(','));
+        core.setOutput('TARGET_BRANCHES', 'master');
         return;
       }
 
-      core.setOutput('TARGET_BRANCH', '');
-      core.setOutput('TRANSFER_LABELS', transferLabels.join(','));
+      // the PR is not coming from a version branch
+      core.setOutput('TARGET_BRANCHES', '');
       return;
     }
 
     core.info(`>>> Target labels found: ${targetLabels.join(', ')}`);
-    let target = '';
 
-    // there was a target branch label present
-    // filter the highest available target number and remove the others from the PR when present
-    if (targetLabels.length > 1) {
-      core.info(`>>> Multiple target labels found.`);
-      targetLabels.sort((a, b) => {
-        const aNum = parseInt(a.match(/\d+/)[0], 10);
-        const bNum = parseInt(b.match(/\d+/)[0], 10);
-        return bNum - aNum;
-      });
-
-      target = targetLabels.shift();
-
-      core.info(`>>> Sorting and setting the highest as 'TARGET_BRANCH' output.`);
-      core.setOutput('TARGET_BRANCH', target);
-
-      // since we have multiple targets we need to add the "needs cherry-pick" label
-      // this makes this workflow de-facto recursive
-      transferLabels.push('needs cherry-pick');
-
-      // add the other targets to the transfer labels
-      transferLabels.push(...targetLabels);
-      core.setOutput('TRANSFER_LABELS', transferLabels.join(','));
-
-      // the others will be removed from the PR
-      core.info(`>>> Removing the other target labels from the PR`);
-      for (const label of targetLabels) {
-        await github.rest.issues.removeLabel({
-          owner,
-          repo,
-          issue_number: pullNumber,
-          name: label,
-        });
-      }
+    // get a list of the original reviewers
+    const reviewers = pr.requested_reviewers.map((reviewer) => reviewer.login);
+    core.info(`>>> Reviewers from original PR: ${reviewers.join(', ')}`);
 
-      core.info(`>>> Creating explanatory comment on PR`);
-      await github.rest.issues.createComment({
+    // the others will be removed from the PR
+    core.info(`>>> Removing target labels from the original PR`);
+    for (const label of targetLabels) {
+      await github.rest.issues.removeLabel({
         owner,
         repo,
         issue_number: pullNumber,
-        body: [
-          `The target branch for the cherry-pick PR has been set to \`${target}\`.`,
-          `Branches that will be created after merging are: ${targetLabels.join(', ')}`,
-          `Thank you!`,
-        ].join('\n\n'),
+        name: label,
       });
-      return;
     }
 
-    core.info(`>>> Removing the "needs cherry-pick" label from the PR`);
+    core.info(`>>> Removing "needs cherry-pick" label from the original PR`);
     await github.rest.issues.removeLabel({
       owner,
       repo,
       issue_number: pullNumber,
-      name: 'needs cherry-pick',
+      name: label,
+    });
```

Comment:
not sure it is necesary to remove them, but probably not an issue in itself 😅

Reply:
its not needed per se, but since the new PR opened by the action will transfer the labels over we would have a problem afterwards